README.md

```diff
@@ -23,54 +18,43 @@ to watch this video to get a better understanding of how it works.
 
 ## Prepare recording in RemotiveCloud
 
-You can try this out in our demo at https://demo.remotivelabs.com but we recommend
-creating a free (or use an existing) account at https://cloud.remotivelabs.com. 
-
-When creating a free account you will get some example drivecycles automatically and
-existing users can click the "import" button in the top right corner under "Recordings".
-
-1. Follow instructions to start a broker and play the recording to the broker, make sure to use "configuration_vss" if you want to use VSS.
-2. Finally press "Go to broker" to open our RemotiveBrokerApp from which you can control the recording being played.
+The first step is to create a [free account at RemotiveCloud](https://cloud.remotivelabs.com) or use an existing account. When creating a free account you will get some sample drivecycles with real vehicle data automatically. Existing users can click the "import" button in the top right corner under "Recordings" to get the same sample drivecycles. 
 
+Next steps:
 
-## Start connect bridge
+- Go to recordings, choose which one to use and prepare that recording for plackback (by clicking the "Prepare for playback"-button). Make sure to use "configuration_vss" if you want to use VSS.
+- Play the recording 
 
-Our ProtoPie Connect bridge is started using RemotiveCLI, read about installation
-https://docs.remotivelabs.com/docs/remotive-cli/installation and details about how to
-use connect is found here https://docs.remotivelabs.com/docs/remotive-cli/cli_man_pages#remotive-connect-protopie
+## Connect to ProtoPie Connect
 
-Once you are playing a recording you can find CLI snippets in our BrokerApp that you can simply
-copy/paste and use directly.
+Our ProtoPie Connect bridge is started using RemotiveCLI, read about [installation of the CLI](https://docs.remotivelabs.com/docs/remotive-cli/installation) and details about how to [connect to ProtoPie Connect](https://docs.remotivelabs.com/docs/remotive-cli/cli_man_pages#remotive-connect-protopie)
 
+In RemotiveCloud you can find a "examples to subscribe to signals from an external application" section where you will get the CLI snippets needed to get you going with connecting to ProtoPie Connect and subscribing to the signals needed to get our sample Pies going. 
 
 In short, this is how it looks
 ```
-remotive connect protopie \
-  --signal vss:Vehicle.Speed \
-  --signal vss:Vehicle.Chassis.SteeringWheel.Angle \
-  --broker-url https://my_cloud_broker \
-  --api-key xxx    
+remotive connect protopie
```

Comment:
\ (backslash) is used to make it possible to copy/paste commands with multiple lines, otherwise it will be multiple commands